client/signup/steps/import-from/wordpress/import-everything/confirm-modal.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from '@automattic/components';
+import { NextButton } from '@automattic/onboarding';
+import { Modal } from '@wordpress/components';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import * as React from 'react';
+
+interface Props {
+	siteSlug: string;
+	onConfirm: () => void;
+	onClose: () => void;
+}
+
+const ConfirmModal: React.FunctionComponent< Props > = ( data ) => {
+	const { __ } = useI18n();
+	const { siteSlug, onConfirm, onClose } = data;
+
+	return (
+		<Modal
+			className={ classnames( 'components-modal-new__frame', 'import__confirm-modal' ) }
+			title={ __( 'Import and replace everything?' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Import and replace everything on this site?' )` *ES Score: 7*